packages/edit-post/src/index.js

```diff
@@ -62,6 +62,7 @@ export function initializeEditor(
 	dispatch( preferencesStore ).setDefaults( 'core', {
 		allowRightClickOverrides: true,
 		editorMode: 'visual',
+		editorTool: 'edit',
```

Comment:
Does it make sense to initialize the default value to edit here? Or we could just change the condition [here](https://github.com/WordPress/gutenberg/blob/3ed56acb9472eebc82d2ebad2d9b9e291f577298/packages/block-editor/src/components/block-tools/use-show-block-tools.js#L43) to `editorMode !== 'navigation'`?

Reply:
I'm on the fence here. On one hand, I don't think it would cause much harm to set the default here, but then we'd probably to add a default for `edit-post` too (and would it also be `'edit'`? probably). On the other hand, fixing the condition to `editorMode !== 'navigation'` seems to more precisely target the underlying issue, which is that hiding the side inserter should be seen as the exception and not the norm — and once you fix that you don't care which editor you're in, nor what state is persisted.

Reply:
Yes agreed, changing the condition for rendering the inserter would be preferable.

Reply:
I think this was the right fix (adding the default) because there could be a lot more places impacted than just the inserter check. That said, I would love for us to avoid these imperative `setDefault` calls somehow at some point, it doesn't feel like a great API. Or maybe the "setDefaults" are not done in the right place, we shouldn't have to duplicate this code everywhere.

Reply:
> That said, I would love for us to avoid these imperative `setDefault` calls somehow at some point, it doesn't feel like a great API. Or maybe the "setDefaults" are not done in the right place, we shouldn't have to duplicate this code everywhere

+1